blog/2024-03-08-structuring-a-large-app.md

```diff
@@ -0,0 +1,391 @@
+---
+author: Masen Furer
+date: 2024-03-08
+title: "Structuring a Large App"
+description: "So your Reflex app is getting large? Here's some advice on how to lay it out."
+image: /blog/structure-large-app.jpg
```

Comment:
nit: we should resize the image, either here or in the blog.py code to make it a consistent size everywhere. Currently it's a bit large.

Reply:
used alek's image